pkg/kando/process_client_test.go

```diff
@@ -5,278 +5,278 @@ import (
 	"context"
 	"encoding/json"
 
-	. "gopkg.in/check.v1"
+	"gopkg.in/check.v1"
```

Comment:
Why in some files, it's imported as
`check "gopkg.in/check.v1"`
but here as `"gopkg.in/check.v1"`?